plugins/GrapesJsBuilderBundle/Model/GrapesJsBuilderModel.php

```diff
@@ -66,13 +60,13 @@ public function addOrEditEntity(Email $email)
             if (isset($data['customMjml'])) {
                 $grapesJsBuilder->setCustomMjml($data['customMjml']);
             }
-        }
 
-        $this->getRepository()->saveEntity($grapesJsBuilder);
+            $this->getRepository()->saveEntity($grapesJsBuilder);
 
-        $customHtml = ArrayHelper::getValue('customHtml', $this->requestStack->getCurrentRequest()->get('emailform'));
-        $email->setCustomHtml($customHtml);
-        $this->emailModel->getRepository()->saveEntity($email);
+            $customHtml = $this->requestStack->getCurrentRequest()->get('emailform')['customHtml'] ?? null;
```

Comment:
@Enc3phale I'm not sure about this. Are we sure that the `customHtml` will be always in the request? If it won't be then it will set the customHtml of the email to null.

Reply:
This assume check  this->requestStack->getCurrentRequest()->request->has('grapesjsbuilder') https://github.com/kuzmany/mautic/blob/fix-yes-no-switch/plugins/GrapesJsBuilderBundle/Model/GrapesJsBuilderModel.php#L57

Then If grapesjsbuilder is in the requests, we assume request is from create/edit page

Reply:
Ok, let's hope it is true 👍